packages/shared/src/utils/shortcode/shortcode.ts

```diff
@@ -38,19 +40,23 @@ export const extractInfoFromShortcode = shortcode => {
     return info_from_shortcode;
 };
 
-export const isHighLow = ({ shortcode = '', shortcode_info = '' }) => {
-    const info_from_shortcode = shortcode ? extractInfoFromShortcode(shortcode) : shortcode_info;
-    return info_from_shortcode && info_from_shortcode.barrier_1 ? !/^S0P$/.test(info_from_shortcode.barrier_1) : false;
+type TIsHighLow = {
+    shortcode?: string;
+    shortcode_info?: {
+        category: string;
+        underlying: string;
+        barrier_1: string;
+        multiplier: string;
+        start_time: string;
+    };
```

Comment:
if shortcode_info is unnecessary property, should we also define all its internal properties like unnecessary ?

Reply:
Thank u @yauheni-kryzhyk-deriv for your revision, this is not unnecessary property, we are using that in one of our components!

Reply:
But about requiring them you are right, I'll change them